app/src/main/java/ru/vssemikoz/newsfeed/models/Category.java

```diff
@@ -15,24 +15,24 @@
 
     private Integer categoryId;
 
-    public static String getCategoryName(Category category){
+    public static String getCategoryName(Category category) {
         return category.toString();
     }
 
-    Category(int stringCategory){
+    Category(int stringCategory) {
         this.categoryId = stringCategory;
     }
 
     @Override
     public String toString() {
-        return MainApplication.getmContext().getString(categoryId);
+        return MainApplication.getContext().getString(categoryId);
     }
 
-    public static String[] getCategoryNameList(){
+    public static String[] getCategoryNameList() {
         Category[] categories = Category.values();
         String[] categoriesName = new String[categories.length];
```

Comment:
Pay more attention to how do you call your variables. Categories name is a name of the set of the categories but you're collecting a string representation of the categories, so it's better to rename it to `categoriesNames` or rename `categories` to `values` and rename `categoriesNames` to `categories`.